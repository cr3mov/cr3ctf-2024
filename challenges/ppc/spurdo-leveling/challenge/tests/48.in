179851483596804809 952407491577644544
3 5242
4 2587
