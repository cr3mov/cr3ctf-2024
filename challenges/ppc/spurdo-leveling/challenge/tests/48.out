294643786415272 147321893207636 0
